packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -332,17 +332,10 @@ To make a column sticky, wrap `Table` with `InnerScrollContainer` and add the fo
 
 To make multiple left-aligned columns sticky:
 
-- wrap Table with InnerScrollContainer
-- add isStickyColumn to all columns that should be sticky
-- add hasRightBorder to the rightmost sticky column
-- add stickyLeftOffset to each sticky column after the first, with a value that equals the combined width - set by stickyMindWidth - of the previous sticky columns
-
-To prevent the default text wrapping behavior and allow horizontal scrolling, all Th or Td cells should also have the modifier="nowrap" property.
-
 - wrap `Table` with `InnerScrollContainer`
 - add `isStickyColumn` to all columns that should be sticky
 - add `hasRightBorder` to the rightmost sticky column
-- add `stickyLeftOffset` to each sticky column after the first, with a value that equals the combined width - set by `stickyMindWidth` - of the previous sticky columns
+- add `stickyLeftOffset` to each sticky column with a value that equals the combined width - set by `stickyMindWidth` - of the previous sticky columns, or 0 for the leftmost column
```

Comment:
If we keep the default of the CSS variables as 0, we could update the last bit to something like, "...of the previous sticky column. The leftmost column should have a value of `0`, which is the default value of this property." Or something to convey that they may not need to pass this property for the leftmost column